### ISSUE TITLE
Arrow alignment check not working with semicolons in a (potential) multiple resources declaration

### ISSUE THREAD

**Issue by shichao-an** [rodjek/puppet-lint#470]
If the resource contains one or more semicolons, even if the syntax is OK (checked with `puppet parser validate`), it will not trigger any arrow alignment warning.

For example, the following code will NOT raise `WARNING: indentation of => is not properly aligned`:

```puppet
class foo {
  file {
    '/etc/rc.d':
      ensure => directory,
      owner => 'root',
      group => 'root',
      mode => '0755';

    '/etc/rc.d/init.d':
      ensure => directory,
      owner => 'root',
      group => 'root',
      mode => '0755';

    '/etc/rc.d/rc0.d':
      ensure => directory,
      owner => 'root',
      group => 'root',
      mode => '0755';
  }
}
```

Or more tentatively, NONE of the following will raise the warning:

```puppet
class bar {
  file {
    '/etc/rc.d':
      ensure => directory,
      owner => 'root',
      group => 'root',
      mode => '0755',
    ;
  }
}
```

```puppet
class bar {
  file {
    '/etc/rc.d':
      ensure => directory,
      owner => 'root',
      group => 'root',
      mode => '0755';
  }
}
```

Only the following version will raise the warning:

```puppet
class bar {
  file {
    '/etc/rc.d':
      ensure => directory,
      owner => 'root',
      group => 'root',
      mode => '0755',
  }
}
```
